common/static/common/js/spec/components/paging_collection.js

```diff
@@ -32,6 +32,44 @@ define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone)
             this.currentPage = currentPage;
             this.start = start;
             return response.items;
+        },
+
+        setPage: function (page) {
```

Comment:
I don't know if I was confused before,  but I see that this is still used. So my question still stands: why are we creating a mock paginated collection rather than just using the true one?